src/main/java/javaslang/jackson/datatype/deserialize/BaseDeserializer.java

```diff
@@ -45,6 +44,13 @@ Object _deserialize(JsonParser jp, JavaType expectedType, DeserializationContext
 
     private Object _deserializeObject(JsonParser jp, JavaType expectedType, DeserializationContext ctx)
             throws IOException, ClassNotFoundException {
+        // TODO (hotfix)
+        if(expectedType != null && !Value.class.isAssignableFrom(expectedType.getRawClass()) && !Tuple.class.isAssignableFrom(expectedType.getRawClass())) {
+            try {
+                JsonDeserializer<?> des = ctx.findRootValueDeserializer(expectedType);
+                return des.deserialize(jp, ctx);
+            } catch (Exception ignore) {}
```

Comment:
Don't worry, it will be deleted :) This exception throws at <code>MixedTest.test5()</code> which also will be deleted